delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/EventsRoutes.scala

```diff
@@ -90,18 +83,17 @@ class EventsRoutes(
                   }
                 },
                 // SSE for events with a given selector within a given project
-                (resolveScopedSelector & projectRef & pathPrefix("events") & pathEndOrSingleSlash) {
-                  (selector, project) =>
-                    concat(
-                      operationName(s"$prefixSegment/$selector/{org}/{proj}/events") {
-                        authorizeFor(project, events.read).apply {
-                          emit(sseEventLog.streamBy(selector, project, offset).attemptNarrow[ProjectRejection])
-                        }
-                      },
-                      (head & authorizeFor(project, events.read)) {
-                        complete(OK)
+                (resolveSelector & projectRef & pathPrefix("events") & pathEndOrSingleSlash) { (selector, project) =>
+                  concat(
+                    operationName(s"$prefixSegment/$selector/{org}/{proj}/events") {
```

Comment:
The operation names can go away because we will keep kamon disabled for this part